src/Support/Svg.php

```diff
@@ -0,0 +1,60 @@
+<?php namespace Winter\Storm\Support;
+
+use Winter\Storm\Exception\ApplicationException;
+use enshrined\svgSanitize\Sanitizer;
+
+/**
+ * A simple wrapper for SVG extraction.
+ *
+ * Under the hood, this uses the "svg-sanitizer" library by Daryll Doyle (https://github.com/darylldoyle/svg-sanitizer)
+ * to sanitize and/or extract the SVG for usage within Winter.
+ *
+ * @author Winter CMS
+ */
+class Svg
+{
+    /**
+     * Extracts and sanitizes SVG code from a given file.
+     *
+     * @param string $path The path to the SVG file.
+     * @param boolean $minify Whether to minify the extracted SVG code.
+     * @return string
+     * @throws ApplicationException If no file, or a malformed SVG, is found at the given path.
+     */
+    public static function extract($path, $minify = true)
```

Comment:
Why Svg::extract() and not Svg::sanitize()?

Reply:
The intention of the utility is to provide the SVG markup for use, either directly in template code or to be populated into a file, hence why I went with `extract`.